frontend/src/main.js

```diff
@@ -21,6 +21,12 @@ const authLink = new ApolloLink((operation, forward) => {
     },
   })
   return forward(operation).map((response) => {
+    if (response.errors && response.errors[0].message === '403.13 - Client certificate revoked') {
+      response.errors[0].message = i18n.t('error.session-expired')
+      store.dispatch('logout', null)
+      if (router.currentRoute.path !== '/login') router.push('/login')
+      return response
+    }
```

Comment:
Whats with `if (!token) return new Error('401 Unauthorized')` ?

Reply:
This one is catched by type-graphql:
` "message": "Access denied! You need to be authorized to perform this action!"`
Do you want to catch this as well to localize it?

Reply:
Its more about you not redirecting in that case.
If the client navigates to a protected page with a broken token, what happens then? Will he be redirected to login already cause of reasons?